_posts/blog/2016-27-06-holacrcy-part-1.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Holacracy - Whatˋs the fuzz about? - Part 1"
+modified:
+categories: blog
+author: peter_krauss
+excerpt: First overview of Holacrcy.
+tags: [Organization, Management, Project]
+date: 2017-06-26T15:39:55-04:00
+---
+
+What is the right way to be happy, productive and innovative as possible? Well, I guess this question 
+pops up in many companies and there are many books out there that gives an answer to this question. 
+However I deeply believe that there isnˋt  -“the one to bind them all”. But some principles are for sure 
+helpful to create a healthy environment - this means for us updudes:
+
+* Independence - you need the freedom to make your own decisions
+* You need to know what you are responsible for
+* You need to know what you can expect from others - so what they are responsible/accountable for
+
+If all this conditions comes true the probability that you finish your task with the highest quality, 
+pretty efficient and even have some fun in the meantime, is pretty high.
+One guide which is on every ones lips in the moment is Holacracy. So we decided to have a closer look 
+and realized that it isn’t the silver bullet (such a surprise), but it includes many tools that we 
+started using to come closer to our Garden Eden.
+
+I will describe in the next sections some key aspects of Holacracy and in the next series of post our 
+experiences with it.
+
+### The CTO is dead - long live the constitution
+
+The classical hierarchical system - one person is in charge and makes every decision - worked for a few 
+hundred years pretty well. It assumes that this person is able to make the best possible decision 
+because she understands every required workflow and still has the big picture overall. This goes true 
+for some assembly lines - like producing pralines. Becomes already unlikely if you think about fully 
+automated, robot driven assembly lines in car production. Is completely impossible in an agile fast 
+changing software development process. 
+What could be the alternative? Anarchy, base democratic decisions or maybe hire an almighty genius who 
+knows everything?
+The answer that Holacracy provides is an accurate set of rules to shift the power to a constitution 
+which is binding for every member of the organization. As like in every other constitution driven 
+culture, this brings the freedom to make your own decisions as long as you stick to the rules. 
+The provocative title of this section could suggest that Holacracy isn’t a hierarchical system - far 
+from it! There is still someone or even a group of people who makes decisions for the whole company, a 
+bit like a government may decide to build an airport, but all the practical choices are made by someone 
```

Comment:
"An example would be a government that decides to build an airport. But all the technical decisions are..."